tests/repository_simulator.py

```diff
@@ -185,6 +189,9 @@ def publish_root(self):
         logger.debug("Published root v%d", self.root.version)
 
     def fetch(self, url: str) -> Iterator[bytes]:
+        """Fetches data from the given url and
```

Comment:
See below for a suggestion to add more content within 80 columns line. (optional, I guess)

```python
        """Fetches data from the given url and returns an Iterator (or yields
        bytes).
        """
```